src/components/utils/src/threads/async_runner.cc

```diff
@@ -88,6 +92,22 @@ void AsyncRunner::AsyncRunnerDelegate::waitForDelegate() {
   }
 }
 
+void AsyncRunner::AsyncRunnerDelegate::clearDelegateQueue() {
+  if (!delegates_queue_.empty()) {
+    std::queue<threads::ThreadDelegate*> queue_to_delete;
```

Comment:
@SNiukalov No appropriate header files included

Reply:
@AByzhynar 
Header file for the std::queue present in async_runner.h

Reply:
@SNiukalov It does not matter. YOu define the entity here - so don't create implicit dependency form header file. Include all headers explicitly

Reply:
@AByzhynar 
I don’t understand why we need duplicates.
This inclusion is present in the header file and also defines the entity.
If you change this header, we will still be forced to change the code in this place. Because it depends on the member of the Delegates_queue_ class.
What is the meaning of this inclusion then?

Reply:
@SNiukalov You defining implicit dependency. This is BAD. You will change something in header e.g. change container from queue to list. What you get is CC file can't be compiled

Reply:
@AByzhynar 
Yes, if change container from queue to list. What you get is CC file can't be compiled.
And if make include to cc  I get is CC file can't be compiled. :) Because deffirents types :)

Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/e62a8ebb4853b56084296b9cb764599bf819d4c9